Added Register page

Jetpabs opened the pull request.

Jetpabs committed:
Add Register page

Jetpabs committed:
Fixed Signup page
Fixed Signup css
Removed Register module

Jetpabs committed:
Updated Signup Page
Updated CSS Module of Signup Page
Added Font Styles for Forms Pages in global.css

Pilner reviewed (changes requested):
Remove unnecessary commented code

`globals.css` - line 123-127
`/signup/page.module.css` - line 59-60
`/signup/page.tsx` - line 77-79

Pilner commented:
also, do the signin/login page

Jetpabs committed:
Added sign in page
Removed unnecessary code
Adjusted labels in register page
Added custom image for the select field

Jetpabs committed:
Fixed link to signup page from login page
